### PR TITLE
1.0.x dev

### PR DESCRIPTION
**Pull request by lordrhodos**
We finally got the code changes for a `1.x` release collected and I would love one last review from you guys for it before merging it into `master` 🙏

### CODE REVIEW

**Comment by felixsand** [comment]
Blocked until it has been tested on a staging and then production system (for example the "SCS") for a while.

Feel free to review it thought :)

**Comment by lordrhodos** [comment]
As @felixsand pointed out, we have tested the branch now for quite some time in production and are going to merge it into master soon to relase a stable `1.0` version of the package. 

@henrikjon @PaulPylkas @felixsand need your review to merge it though ;-)

File: src/ViewInterface.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    public function setLayout(string $script);
+
+    /**
+     * Render output (returns) of the specified view script (with layout if that exists).
+     *
+     * @param string $script Relative script path (without file extension!). Eg. "some-script" or "admin/some-script".
+     * @param bool   $renderLayout
+     * @return string
+     */
+    public function render(string $script, bool $renderLayout = false): string ;
```
Comment (felixsand) [inline src/ViewInterface.php]:
```suggestion
    public function render(string $script, bool $renderLayout = false): string;
```

**Review by felixsand — APPROVED** [review]
I think this looks excellent! 🎉 
Also since this has been previously approved in another PR and also used in both local, testing, staging and production environment for a critical business application (Starweb's ”SCS”) I think this is now to be considered ready for the `v1.0` release

**Pull request merged**